### ISSUE TITLE
Managed & Unmanaged (dynamic) Textures in Libgdx (OpenGL Context Loss)

### ISSUE THREAD

**Issue by RotaruDan** [e-ucm/ead#570]
Related to the creation of dynamic textures in the ShapeRendererProcessor.java:L70 (https://github.com/jtorrente/ead/commit/729f7dc5b5a36d19aea194803d4b12038eac5649) , pr #569 .

We will have problems with this in Android because a texture created from a Pixmap is a dynamic texture that it's not managed by libGDX(because it would be to hard to keep track of every change in a dynamic texture). More info about [Managed & Unmanaged (dynamic) Textures in Libgdx (OpenGL Context Loss)](http://www.badlogicgames.com/wordpress/?p=1073).

This means that if the OpenGL ES context is lost (app goes to background, e.g. because of a call or user presses menu button) when we come back the Texture will be all black (all pixels to 0) so we would have to reload this Texture (from the same Pixmap) in ApplicationListener#resume(...) event.

TL;DR: we will need an aditional system that reloads dynamic textures after a loss of OpenGL ES context, in Android. In order to do this we will have to keep track of all the dynamic textures and their pixmaps.
